README.md

```diff
@@ -172,6 +172,8 @@ Pads the specified integer with leading zeroes to ensure it has at least `places
 
 ## Changelog
 
+1.5.1: The `date` method now correctly returns `null` if the date argument is unparseable and the `def` parameter is explicitly `null`. As was always documented, a default of `undefined` still returns the current date.
```

Comment:
We can say this allows returning null, but an undefined value doesn't return the current date. See #11